ros2topic/test/test_echo_pub.py

```diff
@@ -157,6 +157,8 @@ def message_callback(msg):
                     with launch_testing.tools.launch_process(
                         launch_service, command_action, proc_info, proc_output,
                         output_filter=launch_testing_ros.tools.basic_output_filter(
+                            filtered_patterns=[
+                                r'Waiting for \{[0-7]+\} matching subscription\(s\)\.\.\.'],
```

Comment:
@ivanpauno why the braces? Those go away after `f-string` interpolation.

Reply:
Also why `[0-7]+`? Wouldn't `[0-9]+` be more comprehensive?

Reply:
> Also why [0-7]+? Wouldn't [0-9]+ be more comprehensive?

lol
I've to double-check this, I'm not sure if this filter is doing anything at all.

Reply:
I have delted the filter, as we aren't asserting that the output of the command matches something.